Formula/fzy.rb

```diff
@@ -0,0 +1,17 @@
+class Fzy < Formula
+  desc "Fast, simple fuzzy text selector with an advanced scoring algorithm."
+  homepage "https://github.com/jhawthorn/fzy"
+  url "https://github.com/jhawthorn/fzy/archive/0.8.tar.gz"
+  sha256 "44293ef6a33ef5c9400b9546c580170a2df0ac4342b8fe447cee2af35e241b89"
+  head "https://github.com/jhawthorn/fzy.git"
+
+  def install
+    system "make"
+    system "make", "PREFIX=#{prefix}", "install"
+  end
+
+  test do
+    _, _, stderr = Open3.popen3("#{bin}/fzy -l 0")
+    assert_match /Must be integer in range 3/, stderr.read
```

Comment:
```rb
assert_match "Must be integer in range 3", shell_output("#{bin}/fzy -l 0 2>&1")
```
The above assertion doesn't work with my setup.

Is there a straightforward way to debug the comparison?